src/bin/menu_bar.rs

```diff
@@ -97,6 +102,14 @@ fn build_ui(application: &gtk::Application) {
         window.destroy();
     }));
 
+    quit.add_accelerator(
+        "activate",
+        &accel_group,
+        b'q' as u32,
```

Comment:
IMHO better use http://gtk-rs.org/docs/gdk/enums/key/constant.q.html

Reply:
Thanks for the suggestion. I ended up using `gtk::accelerator_parse` to get access to the `<Primary>` modifier.